pulp_2to3_migration/app/migration.py

```diff
@@ -175,6 +176,11 @@ def create_repo_version(pulp3_repo_name, pulp2_repo):
             pulp2_repo.pulp3_repository_version = new_version
         if not pulp2_repo.pulp3_repository_version:
             pulp2_repo.pulp3_repository_version = pulp3_repo.latest_version()
+        if pulp3_remote:
+            pulp2_repo.pulp3_repository_remote = pulp3_remote
+        # in case importer has not been migrated, for example config was empty
```

Comment:
Can you elaborate, I'm not sure i undertsand the comment.
Why are we doing anyting if importer hasn't been migrated?

Reply:
if pulp3_remote was not provided we automatically try to take the 'native' importer from the repo. i.e. pulp2_repo.pulp2importer. It might happen that the importer was not pre migrated, and therefore we cannot migrate it because we filtered it out( example: it had empty config) in that case if you try accessing pulp2_repo.pulp2importer you will get attribute error.

Reply:
How about "#  pulp2importer might not be migrated, e.g. config was empty"?